src/software/thunderscope/log/g3log_widget.py

```diff
@@ -37,12 +39,19 @@ def __init__(self, buffer_size=10):
         self.layout.addWidget(self.checkbox_widget)
         self.setLayout(self.layout)
 
+        # ignore repeated crash proto
+        self.robot_last_fatal_time_s = []
+        for id in range(MAX_ROBOT_IDS_PER_SIDE):
+            self.robot_last_fatal_time_s.append(0)
+        self.robot_fatal_timeout_s = 5
```

Comment:
Nit: since we want to use this as a constant, lets keep the c++ constant and use all capitals for this

Reply:
updated